src/server/scripts/EasternKingdoms/ScarletEnclave/chapter2.cpp

```diff
@@ -745,11 +745,31 @@ class npc_a_special_surprise : public CreatureScript
     };
 };
 
+enum DevourHumanoid
+{
+    SPELL_DEVOUR_HUMANOID = 53111
+};
+class spell_death_knight_devour_humanoid : public SpellScript
+{
+    PrepareSpellScript(spell_death_knight_devour_humanoid);
+
+    void HandleScriptEffect(SpellEffIndex /* effIndex */)
+    {
+        GetHitUnit()->CastSpell(GetCaster(), SPELL_DEVOUR_HUMANOID, true);
```

Comment:
I think, better use GetEffectValue() instead of hardcoded values. Because this IDs can change in new client version

Reply:
I wonder if such commits are being cherry picked to `master`?

Reply:
if needed some day will be merged.

Reply:
Changed this. Thanks.